src/main/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransaction.java

```diff
@@ -104,8 +113,11 @@ private CaseDetails saveAuditEventForCaseDetails(Event event, CaseType caseType,
         auditEvent.setUserFirstName(idamUser.getForename());
         auditEvent.setCreatedDate(newCaseDetails.getCreatedDate());
         auditEvent.setSecurityClassification(securityClassificationService.getClassificationForEvent(caseType,
-                                                                                                     eventTrigger));
+            eventTrigger));
         auditEvent.setDataClassification(savedCaseDetails.getDataClassification());
+        if (response.getSignificantItem() != null) {
```

Comment:
I believe the null check is not redundant. Setting the significant item even if it's null should be okay.

Reply:
addressed